riot/ble/ble_scan/ble_scan.ipynb

```diff
@@ -115,7 +117,7 @@
    "cell_type": "raw",
    "metadata": {},
    "source": [
-    "TERMLOG=$(pwd)/riot/ble/ble_scan/scan_raw.log make IOTLAB_NODE=<node> -C riot/ble/ble_scan term"
+    "TERMLOG=$(pwd)/riot/ble/ble_scan/scan_raw.log make IOTLAB_NODE=nrf52dk-<id>.<site>.iot-lab.info -C riot/ble/ble_scan term"
```

Comment:
Not sure but maybe node is correct here, since the command that lists the available node returns a list of fqdn names.

Reply:
Will revert!

Reply:
done!